src/main/java/net/sf/jabref/gui/desktop/os/DefaultDesktop.java

```diff
@@ -3,31 +3,37 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
+import net.sf.jabref.external.ExternalFileType;
+
 import java.awt.*;
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+public class DefaultDesktop extends NativeDesktop {
 
-public class DefaultDesktop implements NativeDesktop {
     private static final Log LOGGER = LogFactory.getLog(NativeDesktop.class);
 
+
     @Override
-    public void openFile(String filePath, String fileType) throws IOException {
-        Desktop.getDesktop().open(new File(filePath));
+    public void openFile(String filePath, Optional<ExternalFileType> fileType) throws IOException {
+        openFileWithApplication(filePath, null);
     }
 
     @Override
-    public void openFileWithApplication(String filePath, String application) throws IOException {
+    protected void openFileWithApplication(String filePath, String application) throws IOException {
         Desktop.getDesktop().open(new File(filePath));
     }
 
     @Override
     public void openFolderAndSelectFile(String filePath) throws IOException {
-        File file = new File(filePath);
-        Desktop.getDesktop().open(file.getParentFile());
+        File file = Paths.get(filePath).toAbsolutePath().getParent().toFile();
+        Desktop.getDesktop().open(file);
     }
 
     @Override
-    public void openConsole(String absolutePath) throws IOException {
+    public void openConsoleInDir(String absolutePath) throws IOException {
```

Comment:
I don't like the new method name. For me it is clear that the parameter opens the console inside this directory.